helm-chart/renku-notebooks/values.yaml

```diff
@@ -10,11 +10,31 @@ global:
     domain:
   anonymousSessions:
     enabled: false
+  s3Datasets:
```

Comment:
@ableuler this should be a global value because when s3 dataset support is turned off the UI also needs to know about it so that it does not show the option to mount s3 buckets.

Reply:
should there be an endpoint that lists the capabilities of the notebooks service instead?

Reply:
> an endpoint that lists the capabilities of the notebooks service instead?
Yeah that is another alternative. @ableuler what do you think?

Reply:
Yes, I think that's the cleanest way. Conceptually it's very close to a server option. Should we use the same endpoint?

Reply:
I will check with the UI team what they prefer. I think they directly turn every part of the server options into a UI element that is then used to select options by the user when they launch a session. This setting about the s3 mounting is more an overall setting for the whole delployment that cannot be turned no/off by the users. So because of that I think it may warrant putting into its own endpoint. Perhaps in an endpoint that also shows the current version of the notebook service and other high level info.

Reply:
Yes that's what I had in mind like the version endpoint in the core service

Reply:
Now this is added to the `server_options` endpoint. Confirmed with the UI team they are ok with this.